docs/source/table.rst

```diff
@@ -36,6 +36,12 @@
      - .. image:: https://licensebuttons.net/l/by-sa/4.0/80x15.png
           :target: https://creativecommons.org/licenses/by-sa/4.0
 
+ * - McGill Billboard
```

Comment:
This [table](https://mirdata--466.org.readthedocs.build/en/466/source/quick_reference.html#dataset-quick-reference) looks like it's not getting built - I think it's because this line isn't tabbed over enough!